style.css

```diff
@@ -4,9 +4,38 @@ body {
   width: 100vw;
   height: 100%;
   background-color: #efefef;
+  font-family: 'Poppins', sans-serif;
```

Comment:
- [ ] **[OPTIONAL]** Great job incorporating all the requested changes from the previous reviewer! I just wanted to offer a small optional suggestion **to further enhance the appearance of your project**. It seems like **there might be a small issue causing horizontal movement**, which could be related to certain elements. To quickly identify the problem, I recommend **adding a border to all your elements using the following code:**

   ```
   * {
    border: 1px solid red;
   }
   ```
   By doing this, you'll be able to easily identify any elements that extend beyond the container. Keep up the good work! 🙋‍♂️
   
   ![image](https://github.com/CobbyElsonfx/Portfolio-Website/assets/107952724/f07e49e1-0d83-44c3-bc31-236012398463)